packages/prettier-config/README.md

```diff
@@ -2,14 +2,24 @@
 
 Shared Prettier config for Pandell engineering teams.
 
+For more details and configuration options, see the [prettier documentation](https://prettier.io/docs/en/configuration.html).
```

Comment:
🔍 Really minor, but seems like it should be proper noun casing here.

```suggestion
For more details and configuration options, see the [Prettier documentation](https://prettier.io/docs/en/configuration.html).
```

Reply:
@alexa-pd thank you, updated the link title.